server/routes/tasks.js

```diff
@@ -1,8 +1,36 @@
+const formalizeMultiselectValues = (values) => [values].flat()
+  .filter((it) => !!it)
+  .map((value) => Number(value));
+
 export default (app) => {
   app
     .get('/tasks', async (req, reply) => {
-      const tasks = await app.objection.models.task.query().withGraphJoined('[status, creator, assigned, labels]');
-      await reply.render('tasks/index', { data: { tasks } });
+      const statusIds = formalizeMultiselectValues(req.query.statusIds);
+      const labelIds = formalizeMultiselectValues(req.query.labelIds);
+      const assignedIds = formalizeMultiselectValues(req.query.assignedIds);
+      const tasks = app.objection.models.task.query();
+      if (statusIds.length) {
+        tasks.modify('withStatusIn', statusIds);
+      }
+      if (assignedIds.length) {
+        tasks.modify('withAssignedIn', assignedIds);
+      }
+      tasks.withGraphJoined('[status, creator, assigned, labels]');
```

Comment:
it's kinda pointless to separate this line from line 11

Reply:
done